app/Event/LaravelDispatcherAdapter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Event;
+
+class LaravelDispatcherAdapter implements Dispatcher
+{
+    /** @var  \Illuminate\Events\Dispatcher */
+    private $dispatcher;
+
+    /**
+     * @param \Illuminate\Events\Dispatcher $dispatcher
+     */
+    public function __construct(\Illuminate\Events\Dispatcher $dispatcher)
+    {
+        $this->dispatcher = $dispatcher;
+    }
+
+    /**
+     * @param object|string $event
+     * @param array         $payload
+     */
+    public function dispatch($event, $payload = [])
+    {
+        $this->dispatcher->dispatch($event, $payload, false);
```

Comment:
**Suggestion:** The `dispatch` method in the `LaravelDispatcherAdapter` class does not return anything, which is not consistent with the `Dispatcher` interface it implements. The `dispatch` method in the `Dispatcher` interface is expected to return an array or null. To fix this, you should return the result of the `dispatch` method of the `$dispatcher` object. [bug]
```suggestion
    public function dispatch($event, $payload = [])
    {
        return $this->dispatcher->dispatch($event, $payload, false);
    }
```